CONTRIBUTING.md

```diff
@@ -40,6 +40,24 @@ Remember that contributions are always welcome!
 
 https://github.com/skydive-project/skydive/issues
 
+Making Changes:
+---------------
+
+Make sure that the tests and the linters pass:
+
+make fmt
+make test
+make test.functionals
```

Comment:
for this last one I'm not sure or we should warn the user that it is going to create local network resources